pages/help.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type { NextPage } from 'next'
+import Layout from '../components/layout/Layout'
+import CSVTable from '../components/CSVTable'
+import { ThemeProvider, Typography, Divider } from '@mui/material'
+import { StyledPageContainer } from '../styles/common'
+import { TextContainer } from '../pageStyles/home.styles'
+import { theme } from '../styles/theme'
+
+const Help: NextPage = () => {
+  return (
+        <Layout>
+        <ThemeProvider theme={theme}>
+          <StyledPageContainer>
+            <Typography variant="h3">Help</Typography>
+            <Typography variant="body1">
+            <br />
+              Use this emailsender to quickly send the same email to multiple recipients at once
+              </Typography>
+            <TextContainer>
+              <Typography variant="h5">Steps: </Typography>
+              <ul>
+                <li>
+                  <Typography variant="body1">
+                      Enter the message inside the email. Leave the recipient name in the form of {'$ {name}'} with no spaces
+                      in between. Leave the company name in the form of <br /> {'$ {company}'} with no spaces
+                      in between. The words wrapped in {'$ {}'} will be filled automatically according to the CSV file uploaded.
+                       An example message is shown below.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography variant="body1">
+                      Import a CSV file that includes a table that fills four columns titled --
+                       email, subject, name, company, where email is each recipient&apos;s email, subject is the subject of the email, name is the name of the recipient, company is the name of the company. An example CSV file is shown below.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography variant="body1">
+                      Click on &quot;print final messages&quot; to verify the messages
+                  </Typography>
+                </li>
+                <li>
+                  <Typography variant="body1">
+                      Either click &quot;send preview&quot; to send the previews to your inbox or click &quot;send&quot; to send the emails
```

Comment:
Let's remove the send preview stuff since for now we're just implementing the send